utils/transformers/jw.js

```diff
@@ -1,25 +1,247 @@
+const { urls } = require('../../configs/urls');
```

Comment:
After fixing my mistake in `configs/urls.js`, please revise this to the following:
```suggestion
const { urls } = require('../../configs');
```